casper/src/test/scala/coop/rchain/casper/MultiParentCasperMergeSpec.scala

```diff
@@ -454,4 +424,47 @@ class MultiParentCasperMergeSpec extends FlatSpec with Matchers with Inspectors
       } yield result
     }
   }
+
+  "This spec" should "cover all mergeability cases" in {
+    val allMergeabilityCases = {
+      val events = List(
+        "!X",
+        "!4",
+        "!C",
+        "4X",
+        "4!",
+        "4!!",
+        "PX",
+        "P!",
+        "P!!",
+        "!!X",
+        "!!4",
+        "!!C",
+        "CX",
+        "C!",
+        "C!!"
```

Comment:
Please make it one line, or at most two :D

Reply:
The formatter disagrees

Reply:
One could split it into smaller lists ;) Just FYI - not asking you to do this :)